k8s/k8s.go

```diff
@@ -1609,7 +1609,7 @@ func (k *k8sOps) ValidateStatefulSet(statefulset *apps_api.StatefulSet, timeout
 			return "", true, err
 		}
 
-		pods, err := k.GetStatefulSetPods(statefulset)
+		pods, err := k.GetStatefulSetPods(sset)
```

Comment:
is there any reason we were using **statefulset** instead of updated statefulset  recevied from **GetStatefulSet()** ?
For me UID of stateful set  in context of ClusterMigration required to be updated one

Reply:
I don't think there was any reason there, just a miss.
Good catch.